src/index_cordova.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Meditation+</title>
+  <base href=".">
+
+  <!--
+    Content-Security-Policy:
```

Comment:
Is this comment actually needed?

Reply:
No, was only for having a better overview. I will remove it.